### PR TITLE
Fix: sf5.3/sylius1.9 compatibility

### PR DESCRIPTION
**Pull request by next-sentence**

### CODE REVIEW

File: src/Form/Extension/Api/ProductTypeExtension.php [hunk 1]
```diff
@@ -26,10 +26,15 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
...
     {
         return [
             ProductType::class,
         ];
     }
+
+    public function getExtendedType(): string
+    {
+        return ProductType::class;
+    }
```
Comment (Roshyo) [inline src/Form/Extension/Api/ProductTypeExtension.php:39]: Do we really need to add that ?

Comment (next-sentence) [reply src/Form/Extension/Api/ProductTypeExtension.php:39]: not sure, saw in sylius core something similar so I added it, we can drop this later

Comment (vvasiloi) [reply src/Form/Extension/Api/ProductTypeExtension.php:39]: It's deprecated, remove it.

**Review by Roshyo — APPROVED** [review]

**Comment by romainguerrero** [comment]
Hi, thank you for your work on this plugin ! Do you plan to release a new version with this fix in the next days ?
I have to update this plugin to be able to use in on our upgraded Symfony 5.2 Sylius version, and if I can avoid to be on the `dev-master` branch it would be great 😄

**Comment by vvasiloi** [comment]
@romainguerrero https://github.com/loevgaard/SyliusBrandPlugin/releases/tag/v2.1.5

**Pull request merged**